taskana-adapter-camunda-system-connector/src/test/java/pro/taskana/camunda/camundasystemconnector/acceptance/RetrieveCamundaTaskAccTest.java

```diff
@@ -60,43 +80,47 @@ public void testGetActiveCamundaTasks() {
         expectedTask.setSuspended("false");
         expectedTask.setTaskDefinitionKey("Task_0yogl0i");
 
+        ReferencedTask[] expectedResultBody = new ReferencedTask[] {expectedTask};
+        ResponseEntity<ReferencedTask[]> expectedResult = new ResponseEntity<ReferencedTask[]>(expectedResultBody,
```

Comment:
expectedResult is not used in the class - delete the line

Reply:
changed